packages/aws-cdk-docs/src/_includes.txt

```diff
@@ -49,3 +50,5 @@
 .. So back-to-back entities have a space between them
 
 .. |space| unicode:: 0x20
+
+.. |version| replace:: 0.8.0

```

Comment:
`|cdk-version|` is automatically added by in [conf.py](https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk-docs/src/conf.py#L159)

Reply:
Ok, I'll remove the entry and update the doc topic appropriately.